server/portal/apps/signals/signals.py

```diff
@@ -1,4 +1,4 @@
 from django.dispatch import Signal
 
-# Generic event signal. Not in use as of 6.29.20
-portal_event = Signal(providing_args=['event_type', 'event_data', 'event_users'])
+# Changes for Django 3.*
```

Comment:
@van-go Can you clarify what this comment means?

Reply:
Would something like this work: 
The providing_args argument is deprecated as of Django 3.0